src/webgpu/shader/validation/parse/enable.spec.ts

```diff
@@ -70,36 +70,14 @@ enable f16;`,
     code: `enable subgroups;`,
     pass: true,
   },
-  subgroups_f16_fail: {
-    code: `enable subgroups_f16;`,
-    pass: false,
-  },
-  subgroups_f16_pass1: {
```

Comment:
Maybe I should leave one of these?

Reply:
Yes, maybe we should leave one and have a case to enable both features.